### ISSUE TITLE
`xf:output` used in the UI shouldn't behave like form fields

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#3712]
Right now, in UIs, when we use an `<xf:output>`, the generated HTML contains an `<output tabindex="0" role="textbox">`, and as result the text becomes focusable, as with "Group members" in the screenshot below. In general, this shouldn't be the case. (Often, the `<xf:output>` is not even used not to show a value that is truly dynamic from the perspective of users, but to show a value coming from the resources.)

<img width="502" alt="screen shot 2018-08-31 at 4 16 28 pm" src="https://user-images.githubusercontent.com/106961/44939385-90dddb80-ad39-11e8-9960-e846937445b8.png">

**Comment by ebruchez** [comment]
It's supposed to be the case *only* if the `xf:output` has a label. Maybe it has an empty label in this case?

**Comment by ebruchez** [comment]
It does have a "Role" label, which is hidden.

**Comment by ebruchez** [comment]
Just saw this and it's still the case. The output fields have a hidden external label and that' why they look and feel like fields. It's probably ok, and it's certainly better for accessibility, although in Form Builder a lot is probably not accessible.
